packages/ckeditor5-code-block/src/codeblockcommand.js

```diff
@@ -41,8 +55,11 @@ export default class CodeBlockCommand extends Command {
 	 *
 	 * @fires execute
 	 * @param {Object} [options] Command options.
+	 * @param {String|null} [options.language] Language picked from dropdown.
 	 * @param {Boolean} [options.forceValue] If set, it will force the command behavior. If `true`, the command will apply a code block,
 	 * otherwise the command will remove the code block. If not set, the command will act basing on its current value.
+	 * @param {Boolean} [options.usePreviousLanguageChoice=true] Defines whether or not button will create code block with the same
```

Comment:
```suggestion
	 * @param {Boolean} [options.usePreviousLanguageChoice] If set on `true` and the `options.language` is not specified, the command will apply the previous language (if the command was already executed) when inserting the `codeBlock` element.
```

1. `options.usePreviousLanguageChoice` is marked as optional (`[]`). If not specified, it's a false value (`undefined`).
1. The command itself does not know about the UI, so it does not know what the "button" is.